packages/node/src/login/oidc/oidcHandlers/AuthorizationCodeWithPkceOidcHandler.spec.ts

```diff
@@ -147,5 +149,65 @@ describe("AuthorizationCodeWithPkceOidcHandler", () => {
         mockedStorage.getForUser(oidcOptions.sessionId, "dpop"),
       ).resolves.toBe("false");
     });
+
+    it("emits AUTHORIZATION_REQUEST_STATE event with codeVerifier and state when eventEmitter is provided", async () => {
+      const mockedStorage = mockStorageUtility({});
+      const eventEmitter = new EventEmitter();
+      const authStateSpy = jest.fn();
```

Comment:
```suggestion
      const authStateSpy = jest.fn<(params: { state: string; codeVerifier: string }) => void>();
```
Typing the mock function avoids having a type assertion later.